### DISCUSSION TITLE
Windows Shortcuts not working

### DISCUSSION THREAD

**Discussion by AmansRevenger** [valinet/ExplorerPatcher#743] [Q&A]
Expected behaviour: Win + V should open the clipboard history, Win+ . should open Emoijs etc...

Actual behaviour: Nothing happens

Additional Info :  Is there any way I can get this to work? it is activated in the System Settings > Clipboard etc. ...

**Comment by valinet** [comment]
For me it works, just tried both now. I don't know why it may not work, especially since you're sating you have them enabled.
